i18n/vi.toml

```diff
@@ -165,3 +168,19 @@ other = "Đóng góp với PayPal"
 
 [not_found]
 other = "Trang không tồn tại, vui lòng kiểm tra lại địa chỉ."
+
+[amount_per_year]
+# Ex. $10.000/yr (USD)
+other = "${{ .nb }}/yr (USD)"
```

Comment:
I think the "/yr" (per year) needs the be translated. Maybe "/năm" ? (I'm using Google Translate!)

```suggestion
other = "${{ .nb }}/năm (USD)"
```